src/99-appendices/08-coordinate-systems.md

```diff
@@ -151,6 +151,20 @@ Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
     [ACPC site](https://www.fieldtriptoolbox.org/faq/acpc/) on the FieldTrip
     toolbox wiki.
 
+-   `fsnative`: The subject-specific FreeSurfer T1w space. This corresponds to
+    the original scanner image after it is re-sampled and re-scaled
+    to an isotropic 1 mm 256x256x256 volume in RAS orientation. The origin
+    is wherever the original origin of the ``orig.mgz`` file is. One MUST
+    align the origin to the ``ACPC`` before using this coordinate system.
```

Comment:
the origin must be aligned precisely to the anterior commissure (AC), right? Not to the PC, and also not to an arbitrary point along the AC-PC line.  

AFAIK the reason for referring to ACPC is that the origin is at AC and the y-axis is along the AC-PC line with positive y pointing to anterior (of the subject) and positive z pointing to a mid-hemisphere point that is superior to the AC.

Reply:
Yes you're right. Sorry ACPC alignment is more then what is needed, simply setting the origin at AC should be sufficient and then RAS orientation.